ccd-client/src/main/java/uk/gov/hmcts/cmc/ccd/client/model/CaseDataContent.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.cmc.ccd.client.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Map;
+
+@Data
+@Builder
+public class CaseDataContent {
+    private Event event;
+    private Map<String, JsonNode> data;
```

Comment:
Could we not depend on `JsonNode` here? The point of deserialisation if to not care about the transmission format.

Reply:
And in all other places which use `JsonNode`.